docs/docs/basic-features/segments.md

```diff
@@ -0,0 +1,239 @@
+| `Does not match (!=)` | Trait value is not equal to the rule value                                                                                                               |
+| `% Split`             | Identity is in the percentage bucket. [Learn more](?operators=percent#operator-details)                                                                  |
+| `>`                   | Trait value is greater than the rule value                                                                                                               |
+| `>=`                  | Trait value is greater than or equal to the rule value                                                                                                   |
+| `<`                   | Trait value is less than the rule value                                                                                                                  |
+| `<=`                  | Trait value is less than or equal to the rule value                                                                                                      |
+| `In`                  | Trait value is equal to any element in a comma-separated list (case-sensitive). [Learn more](?operators=in#operator-details)                             |
+| `Contains`            | Rule value is a substring of the trait value                                                                                                             |
+| `Does not contain`    | Rule value is not a substring of the trait value                                                                                                         |
+| `Matches regex`       | Trait value matches the given regular expression                                                                                                         |
+| `Is set`              | Trait value is set for given identity and trait key                                                                                                      |
+| `Is not set`          | Trait value is not set for given identity and trait key                                                                                                  |
+| `SemVer`              | Trait value is compared against the rule value according to [Semantic Versioning](https://semver.org/). [Learn more](?operators=semver#operator-details) |
+
+### Operator details
+
+<Tabs groupId="operators" queryString>
+<TabItem value="in" label="In">
+
+The `In` operator lets you match a trait value against a comma-separated list of values. For example, the segment rule
+value might read `21,682,8345`. This would match against a trait value of `682` but not against a trait value of `683`
+or `834`.
+
+The `In` operator can be useful to build segments that represent a specific set of tenants in your application. For
+example, you could create a segment with the following rule: `tenant_id In tenant_1,tenant_2,tenant_3`
```

Comment:
We have a lot of incoming questions about the segment condition value size limit specifically for the `In` operator.

We should clearly state the condition value size limit somewhere in this doc, and reference it here.

Reply:
Good call, I think I just forgot to add it as part of the rewrite. I'll add that back in

Reply:
Actually that was a false memory 😅 the limits are only mentioned here https://docs.flagsmith.com/system-administration/system-limits

Added a separate section just for the limits in 3de8a58d3c4762eb45329989f56f95a49cf452b5, that should cover all operators and not just `In`.